### PR TITLE
Unesco/issue 171 thumbnail

### PR DESCRIPTION
**Pull request by petrus7**
The css part is not done yet. It is working and displays somehow but its far from done. 

To use widget one should include crop_widget.js, crop_widget.css and jquery on the view where it will be used:

```
<script src="{% static "geonode/js/crop_widget/crop_widget.js" %}/"></script>
<link rel="stylesheet" href="{% static "geonode/css/crop_widget.css" %}">
```

Create img tag (this will be target place for cropped image)
```
   <div>
        <img id="image">
    </div>
```

pass that dom element and the document id to CropWIdget constructor, and initialize widget

```
    <script>
               const image = document.getElementById('image');
               const path = location.pathname.split('/');
               const id = path[path.length - 2];
               const prefetch_url = location.origin + "{{resource.curatedthumbnail.thumbnail_url}}";
               const c_widget = new CropWidget(image,
                      parseInt(
                          id
                       ),
                      '/api/documents/',
                      prefetch_url
                    );
                   c_widget.init();

    </script>
```

Prefetching existing image suffers from ambiguous thumbnail endpoints 

FIX: https://github.com/geosolutions-it/unesco-ihp/issues/171

### CODE REVIEW

**Comment by petrus7** [comment]
# [Codecov](https://codecov.io/gh/petrus7/geonode/pull/1?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@c5b291c`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.


```diff
@@            Coverage Diff            @@
##             master       #1   +/-   ##
=========================================
  Coverage          ?   33.37%           
=========================================
  Files             ?      172           
  Lines             ?    19463           
  Branches          ?     3209           
=========================================
  Hits              ?     6496           
  Misses            ?    12568           
  Partials          ?      399           
```

File: geonode/documents/templates/layouts/doc_panels.html [hunk 1]
```diff
@@ -261,6 +264,24 @@
...
                             <div id="basicInfo" class="container-fluid">
+                                <div>
+                                    <img id="image">
+                                </div>
+                                <script>
+
+                                    const image = document.getElementById('image');
+                                    const path = location.pathname.split('/');
+                                    const id = path[path.length - 2];
+                                    const prefetch_url = location.origin + "{{resource.curatedthumbnail.thumbnail_url}}";
```
Comment (giohappy) [inline geonode/documents/templates/layouts/doc_panels.html]: why don't you use {{ static ... }} here instead of calculating the URL by yourself?

Comment (petrus7) [reply geonode/documents/templates/layouts/doc_panels.html]: im not sure if that is static url. here is an example:
/uploaded/CACHE/images/curated_thumbs/blob_BKSWXbj/0513cd62f2b53208f9a75dc1c48e6766.png

Comment (giohappy) [reply geonode/documents/templates/layouts/doc_panels.html]: I't automatically manahed by static. I told you to give a look to thumbnail_upload.html. You would have seen that static works fine there

Comment (petrus7) [reply geonode/documents/templates/layouts/doc_panels.html]: done

**Pull request closed without merging**